sbin/build.sh

```diff
@@ -719,14 +719,17 @@ generateSBoM() {
   # Add VARIANT
   addSBOMMetadataProperty "${javaHome}" "${classpath}" "${sbomJson}" "VARIANT" "${BUILD_CONFIG[BUILD_VARIANT]^}"
 
-  # Add ALSA 3rd party component
+  # Add GCC component
+  addSBOMComponentFromFile "${javaHome}" "${classpath}" "${sbomJson}" "BuildTools" "Build tools summary" "summary" "${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[TARGET_DIR]}/metadata/dependency_gcc_info.txt"
+
+  # Add ALSA 3rd party component(linux)
   addSBOMComponentFromFile "${javaHome}" "${classpath}" "${sbomJson}" "ALSA" "dependency_version_alsa" "url" "${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[TARGET_DIR]}/metadata/dependency_version_alsa.txt"
 
-  # Add FreeType 3rd party component
+  # Add FreeType 3rd party component (windows + macos)
   addSBOMComponentFromFile "${javaHome}" "${classpath}" "${sbomJson}" "FreeType" "dependency_version_freetype" "url" "${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[TARGET_DIR]}/metadata/dependency_version_freetype.txt"
 
-  # Add FreeMarker 3rd party component
-  addSBOMComponentFromFile "${javaHome}" "${classpath}" "${sbomJson}" "FreeMarker" "dependency_version_freemarker" "url" "${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[TARGET_DIR]}/metadata/dependency_version_freemarker.txt"
+  # Add FreeMarker 3rd party component (openj9)
+  #addSBOMComponentFromFile "${javaHome}" "${classpath}" "${sbomJson}" "" "dependency_version_freemarker" "url" "${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[TARGET_DIR]}/metadata/dependency_version_freemarker.txt"
```

Comment:
commented out code?

Reply:
yeah, dont think for us (non-openj9) we install freemarker. so probably not need to have this call, unless i misunderstand 
 or others are using this script for generating their sbom file